src/utils/customRender.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
```

Comment:
I didn't get the intention behind this file here, maybe add a comment

Reply:
Ohh forgot to add the comment here thanks for pointing it out